llama-index-integrations/vector_stores/llama-index-vector-stores-milvus/llama_index/vector_stores/milvus/base.py

```diff
@@ -276,11 +276,11 @@ def __init__(
             **kwargs,  # pass additional arguments such as server_pem_path
         )
         # Delete previous collection if overwriting
-        if overwrite and collection_name in self.client.list_collections():
+        if overwrite and collection_name in self._milvusclient.list_collections():
```

Comment:
Any reason why using the raw field instead of the default property?

Reply:
typechecking?

Reply:
I thought both are ok. But after thinking about it, I think self.client is a more concise way. So I have updated it